config/initializers/sentry.rb

```diff
@@ -1,10 +1,6 @@
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = Rails.application.config.x.sentry_dsn
-  config.environments = %w[production staging]
-  config.current_environment = Rails.env
+  config.enabled_environments = %w[production staging]
+  config.environment = Rails.env
   config.release = ENV.fetch('BUILD_HASH', nil)
-  config.async = lambda { |event|
-    SentryJob.perform_later(event)
-  }
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
```

Comment:
filter_parameters only contains `password`, which is in the default sanitation list (https://github.com/getsentry/sentry-ruby/blob/ca597ad12353e0417f2ab9807c35e06b53116302/sentry-raven/lib/raven/processor/sanitizedata.rb#L7)